lib/central.js

```diff
@@ -44,7 +46,11 @@ Central.prototype.connect = function(callback) {
     }
   }.bind(this));
 
-  this.bleConnect.startScanning();
+  this.bleConnect.on("stateChange", function(state) {
+    if (state === "poweredOn") {
+      that.bleConnect.startScanning();
+    }
+  });
```

Comment:
Can we keep using the `Function#bind(this)` syntax from previous callbacks (`discover` listener, etc)?